src/test/java/com/google/api/codegen/gapic/testdata/py/python_library.baseline

```diff
@@ -5276,9 +5269,7 @@ class TestLibraryServiceClient(object):
 
     def test_publish_series(self):
         # Setup Expected Response
-        book_names_element = 'bookNamesElement1491670575'
-        book_names = [book_names_element]
-        expected_response = {'book_names': book_names}
+        expected_response = {}
```

Comment:
I am confused by this. The change causes a change to the unit test, but no corresponding change to actual code. (I would have expected no change at all).

This appears to be true in PHP, Python, and Ruby, and as best as I can tell it indicates that (1) there is no breaking change and (2) that test is bogus.

Reply:
What about the change in the [method descriptors](https://github.com/googleapis/gapic-generator/pull/2488/files#diff-f7e1b0b3f703b5c8ece5073db3bce027L3507)?